packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20191021.twig

```diff
@@ -0,0 +1,521 @@
+            auf der Website oder vom Nutzer getätigte Mausbewegungen). Die bei der Analyse erfassten Daten werden an
+            Google weitergeleitet.</p>
+        <p>Die reCAPTCHA-Analysen laufen vollständig im Hintergrund. WebseitenbesucherInnen werden nicht darauf
+            hingewiesen, dass eine Analyse stattfindet.</p>
+        <p>Die Datenverarbeitung erfolgt auf Grundlage von Art. 6 Abs. 1 lit. f DSGVO. Der Webseitenbetreiber hat ein
+            berechtigtes Interesse daran, seine Webangebote vor missbräuchlicher automatisierter Ausspähung und vor SPAM
+            zu schützen.</p>
+        <p>Weitere Informationen zu Google reCAPTCHA sowie die Datenschutzerklärung von Google entnimmst Du folgenden
+            Links: <a href="https://policies.google.com/privacy?hl=de">https://policies.google.com/privacy?hl=de</a> und
+            <a href="https://www.google.com/recaptcha">https://www.google.com/recaptcha</a>.</p>
+        <h4>Registrierung auf dieser Website</h4>
+        <p>Du kannst dich auf unserer Website registrieren, um zusätzliche Funktionen auf der Seite zu nutzen. Die dazu
+            eingegebenen Daten verwenden wir nur zum Zwecke der Nutzung des jeweiligen Angebotes oder Dienstes, für den
+            Du dich registriert hast. Die bei der Registrierung abgefragten Pflichtangaben müssen vollständig angegeben
+            werden. Andernfalls werden wir die Registrierung ablehnen.</p>
+        <p>Für wichtige Änderungen etwa beim Angebotsumfang oder bei technisch notwendigen Änderungen nutzen wir die bei
+            der Registrierung angegebene E-Mail-Adresse, um Dich auf diesem Wege zu informieren.</p>
+        <p>Die Verarbeitung der bei Registrierung angegebenen Daten geschieht auf Grundlage des Art. 6 Abs. 1 lit. a
+            DSGVO mit Deiner Einwilligung. Du kannst eine von Dir erteilte Einwilligung jederzeit widerrufen. Dazu
+            reicht eine formlose Mitteilung per E-Mail an uns. Die Rechtmäßigkeit der bereits erfolgten
+            Datenverarbeitung bleibt vom Widerruf unberührt.</p>
+        <p>Die bei der Registrierung erfassten Daten werden von uns gespeichert, solange Du auf unserer Website
+            registriert sind und werden anschließend gelöscht. Gesetzliche Aufbewahrungsfristen bleiben unberührt.</p>
+        <h4>Aktivitätsdaten</h4>
+        <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, werden die folgenden Aktivitäten
+        in unserer Datenbank gespeichert: Artikelerstellung, Artikeländerung, Artikelreview. Serlo erfasst jeweils deinen
+        Nutzernamen und Zeitpunkt der Aktivität. Die Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer
+        Onlinecommunity zu verbessern und sicher zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung
+        der Daten erfolgt über Monitoring- und Analyseprogrammen, welche von Serlo in der Google Cloud betrieben wird
```

Comment:
Doesn't matter here. We already stated that we use Google Cloud. And as long as no external entity is used for analyzing specifically that data, you don't need to repeat that everywhere.